### ISSUE TITLE
Feature request: ability to change Lutris locale (language) by clicking in UI (not coding in terminal)

### ISSUE THREAD

**Issue by vrement** [lutris/lutris#5789]
I just learned that a most basic thing is missing from Lutris: the language setting in Lutris is to control the language of a game, not Lutris itself. This is not only confusing but also a very basic thing lacking.

You could run Lutris with the LANG environment variable to control Lutris itself with opening terminal and then coding: LANG=pt_BR lutris but most users don't even know what terminal is nevermind coding this stuff just to change language.

**Comment by yodog** [comment]
yeap. i just installed lutris and for some reason it decided the language that i should use instead of the standard, globally used... english.

go figure...

**Comment by vrement** [comment]
Yes the problem still exists. A simple dropdown button to change the language would suffice. Things are once again here made unnecessarily cumbersome without even a sane way to fix.
